From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@NataliaL__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-nalubs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

joaoma commented:
this article is actually mine. #sign-off

Commit:
Added mention of portal option.

v-geberr committed:
change date

PRMerger13 committed:
Merge pull request #34683 from Brenduns/patch-1

Update azure-stack-asdk-release-notes.md

tynevi committed:
Merge pull request #33661 from sptramer/go/landing-tab

Add Go SDK links to landing pages.

PRMerger13 committed:
Merge pull request #34661 from kgremban/mar6_iotcore

Changes to IoT Edge IoT Core for code update

Brenduns committed:
Update azure-stack-update-1802.md

Update for 1802 release notes. Working around issue https://mseng.visualstudio.com/CSI/_workitems/edit/1200443

PRMerger16 committed:
Merge pull request #34685 from mrbullwinkle/mrb_03_06_avail

fix metadata

Commit:
Added VNet address space text.

Commit:
Fix

Commit:
Fix.

Brenduns committed:
Update azure-stack-mysql-resource-provider-deploy.md

SQL Provider version update.

Commit:
Fix

Brenduns committed:
Update azure-stack-sql-resource-provider-deploy.md

SQL RP version update for 1802.

Commit:
Fix

Commit:
Fix

PRMerger18 committed:
Merge pull request #34690 from Brenduns/patch-2

Update azure-stack-update-1802.md

CarlRabeler committed:
Update sql-database-high-availability.md

kgremban committed:
Changing author of LBS docs to simplify public prs

CarlRabeler committed:
Update sql-database-technical-overview.md

neilpeterson committed:
inital doc outline

douglaslMS committed:
Cleaned up recently added section.

CarlRabeler committed:
Update sql-database-designing-cloud-solutions-for-disaster-recovery.md

Lisaco88 committed:
Merge pull request #34649 from CarlRabeler/rmi-test

Real

CarlRabeler committed:
Update sql-database-disaster-recovery-strategies-for-applications-with-elastic-pool.md

tdykstra committed:
convert to png

yujhongmicrosoft committed:
Update documentation-government-cognitiveservices.md

PRMerger19 committed:
Merge pull request #34692 from Brenduns/patch-3

Update azure-stack-mysql-resource-provider-deploy.md

CarlRabeler committed:
Merge branch 'release-managed-instance' into rmi-borko

TerryLanfear committed:
removing preview

PRMerger20 committed:
Merge pull request #34693 from Brenduns/patch-4

Update azure-stack-sql-resource-provider-deploy.md

TerryLanfear committed:
removing preview

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into new-acs

Commit:
Merge pull request #34691 from jimdial/patch-88

Added VNet address space text.

wesmc7777 committed:
Add WAR file deployment

CarlRabeler committed:
toc

twooley committed:
[Azure AD B2b] Updated statement about Dynamics support

douglaslMS committed:
Cleanup.

douglaslMS committed:
Fixed an Acrolinx typo.

PRMerger10 committed:
Merge pull request #34700 from yujhongmicrosoft/patch-10

Update documentation-government-cognitiveservices.md

PRMerger10 committed:
Merge pull request #33637 from rothja/lifecycle2

Updating lifecycle info in SQL VM overview

douglaslMS committed:
Added note about FIPS encryption issue per email thread.

PRMerger12 committed:
Merge pull request #34702 from kgremban/mar6_lbsauthor

Changing author of LBS docs to simplify public prs

PRMerger13 committed:
Merge pull request #34703 from twooley/twb2bissue

[Azure AD B2B] Updated statement about Dynamics support

Ja-Dunn committed:
Merge pull request #34560 from tdykstra/wjcs

add webjobs sdk reference

v-kents committed:
add third party notice file

PRMerger14 committed:
Merge pull request #34704 from v-geberr/0306-pub-date-1223

change date

wesmc7777 committed:
remove some git deployment comments

PRMerger16 committed:
Merge pull request #34699 from douglaslMS/adf-custom-activity

Cleaned up recently added section.

Ja-Dunn committed:
Merge pull request #33633 from rothja/crosslinkingfixes

Adding cross-linking and a new tip to the pricing guidance page

CarlRabeler committed:
toc

douglaslMS committed:
Updated PowerShell section per email thread.

douglaslMS committed:
Fixed a typo.

CarlRabeler committed:
Update monitoring-stream-diagnostic-logs-to-event-hubs.md

CarlRabeler committed:
Update sql-database-managed-instance-migrate.md

CarlRabeler committed:
Update sql-database-managed-instance-transact-sql-information.md

v-geberr committed:
fix for unreferenced intent

kgremban committed:
Corrections explaining how the Edge ML model works

edmacauley committed:
verbage change

edmacauley committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

wesmc7777 committed:
Referencing Eclipse for handling the sample

Ja-Dunn committed:
Merge pull request #34696 from j-martens/master

Trying again to get a clean build

Commit:
Merge pull request #34706 from douglaslMS/adf-runtime-fips

Added note about FIPS encryption issue per email thread.

buffalo-sauce-rules committed:
Update routing-coverage.md

buffalo-sauce-rules committed:
Update routing-coverage.md

MGoedtel committed:
updated VM ext version table

buffalo-sauce-rules committed:
Update routing-coverage.md

wesmc7777 committed:
minor fix

buffalo-sauce-rules committed:
Update routing-coverage.md

buffalo-sauce-rules committed:
Update routing-coverage.md

neilpeterson committed:
updates

markgalioto committed:
fixing the service-fabric TOC.yml issue in this PR

MGoedtel committed:
updated Windows agent quickstart

buffalo-sauce-rules committed:
Update render-coverage.md

tdykstra committed:
make one page

Ja-Dunn committed:
Merge pull request #30030 from CarlRabeler/zones

[sql db] zone redundancy

buffalo-sauce-rules committed:
Update render-coverage.md

kgremban committed:
Fixed anchor link

buffalo-sauce-rules committed:
Update zoom-levels-and-tile-grid.md

edmacauley committed:
changing pre-req article

edmacauley committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

edmacauley committed:
changing pre-req article

PRMerger14 committed:
Merge pull request #34604 from markgalioto/mar52018

fixing MABS article with issue from Support

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#5321

Adds sentence to warn that existing SSH key-pair is overwritten if defaults are used.

v-geberr committed:
add LUIS app location to docs

v-alje committed:
Resolving merge conflict

PRMerger15 committed:
Merge pull request #34718 from MGoedtel/UpdateLALinuxExt36

updated VM ext version table

v-geberr committed:
modified ms.date

neilpeterson committed:
updates:

PRMerger17 committed:
Merge pull request #34723 from iainfoulds/patch-14

[VM] Fixes MicrosoftDocs/azure-docs#5321

Ja-Dunn committed:
Merge pull request #34716 from kgremban/mar6_mlmodel

Corrections explaining how the Edge ML model works

Ja-Dunn committed:
Merge pull request #34721 from tdykstra/wjcs

consolidate quick reference pages

martincoetzer committed:
Update active-directory-conditional-access-technical-reference.md

Added information about chrome support in Windows 8.1 and Windows 7

ktoliver committed:
Merge pull request #34675 from CarlRabeler/rmi-borko

borko

itechedit committed:
Update active-directory-saas-esalesmanagerremix-tutorial.md

v-geberr committed:
switch out "check" for "select"

PRMerger19 committed:
Merge pull request #34698 from neilpeterson/pvc-files-update

[AKS] - Azure files PVC update

v-alje committed:
Merge pull request #34724 from MicrosoftDocs/FromPublicRepo

From public repo

apwestgarth committed:
Updated docs to support update 1 release

wesmc7777 committed:
Adding updates to help customers notice differences in Java app pathes

daveba committed:
fixing redirects and adding toc file for MSI

apwestgarth committed:
Additional links for the update documentation

v-geberr committed:
note where in the code the text is added

apwestgarth committed:
Added missing image

apwestgarth committed:
Adding missing certificates screenshot

apwestgarth committed:
Fixed get certificates link

CarlRabeler committed:
[sql db] managed instance threat detection

wesmc7777 committed:
minor wording update

gunjanj committed:
Update to Interop section

- Added FCI and Azure Info Protection details

GitHubber17 committed:
Merge pull request #34701 from TerryLanfear/new-acs

New acs

GitHubber17 committed:
Update luis-csharp-tutorial-build-bot-framework-sample.md

CarlRabeler committed:
Update sql-database-high-availability.md

apwestgarth committed:
Fixed file link to certificate documentation

apwestgarth committed:
Fixed certificate information link

CarlRabeler committed:
Update sql-database-high-availability.md

ShawnJackson committed:
Merge pull request #34582 from itechedit/active-directory-saas-esalesmanagerremix-tutorial

edit pass: active-directory-saas-esalesmanagerremix-tutorial

PRMerger17 committed:
Merge pull request #34726 from martincoetzer/patch-1

Update active-directory-conditional-access-technical-reference.md

GitHubber17 committed:
Merge pull request #34714 from v-geberr/0306-tut-csharp-bot

fix for unreferenced intent

zhiweiwangmsft committed:
risky IP report

CarlRabeler committed:
Update sql-database-managed-instance.md

CarlRabeler committed:
Update sql-database-managed-instance.md

Commit:
Merge pull request #34732 from CarlRabeler/patch-9

Update sql-database-high-availability.md

apwestgarth committed:
Updated with download links and correct date for publish

CarlRabeler committed:
Update sql-database-managed-instance-migrate.md

Commit:
Merge pull request #34735 from zhiweiwangmsft/patch-16

risky IP report

CarlRabeler committed:
Update sql-database-managed-instance-transact-sql-information.md

Commit:
Merge pull request #34736 from CarlRabeler/patch-10

Update sql-database-managed-instance.md

tynevi committed:
Merge pull request #34519 from daveba/moving-msi-files

[Azure AD MSI] Moving msi content to its own subdir and create redirects

Commit:
Merge pull request #34737 from CarlRabeler/patch-11

Update sql-database-managed-instance.md

apwestgarth committed:
Changes to improve doc and fix spelling

CarlRabeler committed:
Update toc.yml

PRMerger10 committed:
Merge pull request #33874 from klaaslanghout/patch-13

we fixed a bug/updated to note the version

rjagiewich committed:
Merge pull request #34730 from CarlRabeler/rmi-threat

[sql db] managed instance threat detection

buffalo-sauce-rules committed:
Update render-coverage.md

Lisaco88 committed:
Merge pull request #34663 from edmacauley/localworking

new pr for sql server to mi

buffalo-sauce-rules committed:
Update render-coverage.md

CarlRabeler committed:
Update sql-database-managed-instance-threat-detection.md

PRMerger15 committed:
Merge pull request #34739 from CarlRabeler/patch-13

Update sql-database-managed-instance-transact-sql-information.md

buffalo-sauce-rules committed:
Update routing-coverage.md

buffalo-sauce-rules committed:
Update zoom-levels-and-tile-grid.md

PRMerger16 committed:
Merge pull request #34738 from CarlRabeler/patch-12

Update sql-database-managed-instance-migrate.md

buffalo-sauce-rules committed:
Update render-coverage.md

Lisaco88 committed:
Merge pull request #34740 from CarlRabeler/patch-14

Update toc.yml

CarlRabeler committed:
Update sql-database-aad-authentication.md

GitHubber17 committed:
Merge pull request #34361 from orestis-ms/master

Documentation for MongoDB Indexing

PRMerger19 committed:
Merge pull request #34741 from CarlRabeler/patch-15

Update sql-database-managed-instance-threat-detection.md

CarlRabeler committed:
Update sql-database-managed-instance-tutorial-portal.md

ealsur committed:
Updating fork

Updating fork

CarlRabeler committed:
Update sql-database-managed-instance-migrate.md

Commit:
Merge pull request #34742 from CarlRabeler/patch-16

Update sql-database-aad-authentication.md

CarlRabeler committed:
Update sql-database-managed-instance-tutorial-portal.md

ealsur committed:
Adding 1.3.0 release notes

CarlRabeler committed:
Update sql-database-managed-instance-threat-detection.md

CarlRabeler committed:
Update sql-database-managed-instance.md

CarlRabeler committed:
Update toc.yml

Commit:
Merge pull request #34745 from CarlRabeler/patch-18

Update sql-database-managed-instance-migrate.md

gunjanj committed:
Feedback incorporated

Minor updates

PRMerger10 committed:
Merge pull request #34747 from CarlRabeler/patch-19

Update sql-database-managed-instance-threat-detection.md

PRMerger12 committed:
Merge pull request #34743 from CarlRabeler/patch-17

Update sql-database-managed-instance-tutorial-portal.md

PRMerger12 committed:
Merge pull request #34748 from CarlRabeler/patch-20

Update sql-database-managed-instance.md

yujhongmicrosoft committed:
Update documentation-government-services-securityandidentity.md

PRMerger14 committed:
Merge pull request #34731 from gunjanj/patch-9

Update to Interop section

mimckitt committed:
Removing Duplicate Sentences

It says "This will also ensure all console messages are sent to the first serial port, which can assist Azure support with debugging issues" twice: once before the example once after.

Removed the second so there is no more duplicate

v-geberr committed:
fix tutorial with new portal blade image and wording

spelluru committed:
fixed images

Commit:
deprecated link removed

PRMerger13 committed:
Merge pull request #34671 from cynthn/patch-66

Update tutorial-iis-sql.md

PRMerger17 committed:
Merge pull request #34760 from spelluru/nhubwphone-0306

fixed images

rboucher committed:
Fix title typo.

rboucher committed:
Update for acrolinx

Lisaco88 committed:
Merge pull request #34749 from CarlRabeler/patch-21

Update toc.yml

wesmc7777 committed:
sync date with the publish date

femsulu committed:
[ForumTeam] Guthub Issue: #5277

Changed "Standard" & "Premium" to " Basic" and "Enterprise" respectively 
Source: https://github.com/MicrosoftDocs/azure-docs/issues/5277

Lisaco88 committed:
Fixed ms.date

PRMerger18 committed:
Merge pull request #34762 from femsulu/patch-1

[ForumsTeam] Github Issue: #5277

PRMerger18 committed:
Merge pull request #34755 from micahmckittrick/patch-3

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5365

Lisaco88 committed:
Merge pull request #34757 from v-geberr/0306-tut-node-web-bot

fix tutorial with new portal blade image and wording

Lisaco88 committed:
Deleted repeated word

Lisaco88 committed:
Merge pull request #34546 from jinzh-azureiot/master

create "zoom levels and tile grid" doc

Lisaco88 committed:
Fixed 2 typos

Lisaco88 committed:
Fixed typos

PRMerger14 committed:
Merge pull request #34761 from rboucher/patch-13

Fix title typo.

PRMerger14 committed:
Merge pull request #34758 from cjgronlund/cog-serv-link

Cog Services: deprecated link removed

Lisaco88 committed:
Merge pull request #34598 from wesmc7777/java_web_app_app_svc_linux

Java web app app svc linux

Commit:
link updates

Commit:
links to be  moved to new section removed from old section

Commit:
consistent naming

PRMerger18 committed:
Merge pull request #34763 from cjgronlund/cog-serv-link

Cognitive Services link updates

rayne-wiselman committed:
Update .openpublishing.redirection.json

linda33wj committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

linda33wj committed:
Fix MongoDB typo

linda33wj committed:
Reword gateway to IR in format topic

PRMerger10 committed:
Merge pull request #34768 from msvijayn/gartner-vijayn

Metrics from Log - Limited Public Preview

PRMerger12 committed:
Merge pull request #34767 from linda33wj/master

Update ADF copy content

apwestgarth committed:
Fixed numbering and layout issues

apwestgarth committed:
removed trailing ] certificate image

rayne-wiselman committed:
Update toc.yml

Commit:
Introducing POA 1.2.1 for windows.
Added sfpkg file for customers who'd been using ARM based deployment.

PRMerger13 committed:
Merge pull request #34415 from cephalin/deploy

git-deploy rewrite

Commit:
Fix.

Commit:
Merge branch 'master' of https://github.com/novino/azure-docs-pr

Commit:
Merge pull request #34686 from jimdial/accelerated-networking

Added mention of portal option.

Commit:
update per customer feedback

Commit:
Updates to change UI in the app

Commit:
Merge pull request #1 from danimir/danimir-patch-1

Updates to change UI in the app

PRMerger18 committed:
Merge pull request #34771 from Blackmist/freshness

update per customer feedback

msvijayn committed:
Link to Azure Monitor doc

Commit:
Update to the screenshot due to the app UI changes

Update to the screenshot due to the app UI changes

Commit:
Merge pull request #2 from danimir/danimir-patch-1

Update to the screenshot due to the app UI changes

Commit:
Changed accept to except.

Commit:
Merge pull request #34773 from jimdial/patch-89

Changed accept to except.

PRMerger12 committed:
Merge pull request #34751 from yujhongmicrosoft/patch-12

Update documentation-government-services-securityandidentity.md

PRMerger15 committed:
Merge pull request #34775 from msvijayn/gartner-vijayn

Link to Azure Monitor doc

bsiva committed:
ReiserFs and Multipath support statement update

bsiva committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

HeidiSteen committed:
metadata update

PRMerger14 committed:
Merge pull request #34746 from ealsur/master

Updating Cosmos DB Change Feed SDK 1.3.0 Release notes

PRMerger15 committed:
Merge pull request #34784 from bsiva/master

ASR VMware to Azure storage support update

v-alje committed:
Merge pull request #34665 from MicrosoftDocs/release-managed-instance

Release managed instance

douglaslMS committed:
Fixed quotes in PS script sample per email thread.

Lisaco88 committed:
Merge pull request #34782 from HeidiSteen/heidist-ta

metadata update

rjagiewich committed:
Merge pull request #34774 from danimir/master

Updates to screenshots due to UI changes in the app

MikeDodaro committed:
Change names in samples

PRMerger18 committed:
Merge pull request #34713 from douglaslMS/adf-ssisir-vnet

Updated PowerShell section per email thread.

Commit:
Misspelling

Commit:
Minor doc correction for POA 1.2.1 release

Commit:
Merge pull request #34785 from MikeDodaro/legalNamesEntity

Change names in samples

Commit:
quickfix08

martincoetzer committed:
Update active-directory-conditional-access-technical-reference.md

light formatting update

Brenduns committed:
Update azure-stack-app-service-deploy.md

Correcting the indents for the procedure.

Commit:
Update for portal UI change.

CarlRabeler committed:
links

PRMerger13 committed:
Merge pull request #34769 from novino/master

POA 1.2.1 release for windows

rjagiewich committed:
Merge pull request #34492 from rayne-wiselman/master

rename VMware to Azure files

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger15 committed:
Merge pull request #34791 from Jann-Skotdal/master

Speech Translator API Misspelling

adegeo committed:
Revert "toc delete"

This reverts commit ae389e2538aa4c77d96e568d518afc5036a7da8e which
deleted a TOC file!

tynevi committed:
Merge pull request #34709 from v-kents/item_1197224

add third party notice file

Lisaco88 committed:
Merge pull request #34727 from apwestgarth/master

Updated docs to support update 1 release

ausfeldt committed:
add 1.20.0 to release note again

@mimig1

rwike77 committed:
fixing metadata

PRMerger18 committed:
Merge pull request #34788 from martincoetzer/patch-2

Update active-directory-conditional-access-technical-reference.md

tynevi committed:
Merge pull request #34800 from Thraka/sf-restoretoc

Revert "toc delete"

Commit:
Merge pull request #34799 from ausfeldt/patch-14

add 1.20.0 to release note again

Commit:
Merge pull request #34801 from rwike77/metadata

fixing metadata

v-alje committed:
Merge pull request #34797 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #34790 from MarkusVi/quickfix08

quickfix08

Commit:
Merge pull request #34793 from CarlRabeler/MI

links

Commit:
Merge pull request #34789 from jimdial/patch-91

Update for portal UI change.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
